app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,22 @@
+class LessonsController < ApplicationController
+  def create
+    @category = Category.find_by_id(params[:category_id])
+    @user = User.find_by_id(params[:user_id])
```

Comment:
@CharlotteDS You can access `current_user` directly which already is a `User` instance